bundles/org.eclipse.ui.workbench/eclipseui/org/eclipse/ui/internal/misc/ExternalProgramImageDescriptor.java

```diff
@@ -26,7 +25,7 @@
  */
 public class ExternalProgramImageDescriptor extends ImageDescriptor {
 
-	public Program program;
+	private Program program;
```

Comment:
Can RCPTT get a better `toString()` or an adapter to be able to introspect this?

Reply:
Yes implemention `IAdaptable` would be the right approach here to access internal data structure.

Reply:
Done in
- https://github.com/eclipse-platform/eclipse.platform.ui/pull/2991